core/templates/pages/learner-dashboard-page/modal-templates/remove-activity-from-learner-dashboard-modal.controller.spec.ts

```diff
@@ -58,22 +58,24 @@ describe('Remove Activity From Learner Dashboard Modal Controller',
         });
       }));
 
-      it('should init the variables', function() {
-        expect($scope.sectionNameI18nId).toEqual(sectionNameI18nId);
-        expect($scope.subsectionName).toEqual(subsectionName);
-        expect($scope.activityTitle).toEqual(activity.title);
-      });
-
-      it('should remove activity', function() {
-        $httpBackend.expectDELETE(
-          '/learnerplaylistactivityhandler/exploration/1').respond(200);
-        $scope.remove();
-
-        $httpBackend.flush();
-        expect($uibModalInstance.close).toHaveBeenCalled();
-        $httpBackend.verifyNoOutstandingExpectation();
-        $httpBackend.verifyNoOutstandingRequest();
-      });
+      it('should initialize $scope properties after controller is initialized',
+        function() {
+          expect($scope.sectionNameI18nId).toEqual(sectionNameI18nId);
+          expect($scope.subsectionName).toEqual(subsectionName);
+          expect($scope.activityTitle).toEqual(activity.title);
+        });
+
+      it('should remove activity from backend when clicking remove',
```

Comment:
Clicking remove button probably is better. Always add what type of element gets the click to the name

Reply:
Done.